bin/taxon_start_service.py

```diff
@@ -49,7 +82,8 @@ def main():
         #signal.signal(signal.SIGHUP, driver.reload_config())
 
         try:
-            driver.start_service(port=args.port)
+            print "starting service on port " + str(service_port)
+            driver.start_service(services=services,port=service_port,host='')
```

Comment:
For listening on all interfaces there is something we can do when the twisted server is instantiated, so I can help here if that isn't working as expected.